_pages/features/notice-and-comment.md

```diff
@@ -12,12 +12,9 @@ agencies to more easily manage and contextually organize the comments coming
 in. The pilot project launches in mid-July with the Environmental Protection
 Agency’s eManifest team.
 
+ 
+## Sprint demo playlist (11 videos)
 
-## Links to demo videos
-
-* [Sprint 1](https://www.youtube.com/watch?v=w9LCTExyC3A)
-* [Sprint 2](https://www.youtube.com/watch?v=vH6UznuKyu0)
-* [Sprint 3 - First Wireframes!](https://www.youtube.com/watch?v=7aEjjk-JrKg)
-* [Sprint 4 - More Wireframes!](https://www.youtube.com/watch?v=wFEDa7CsQAo)
-* [Sprint 5 - Wireframes &amp; Working Code!](https://www.youtube.com/watch?v=h8DyhPOgUt4)
-
+<iframe width="560" height="315"
```

Comment:
Could you make the youtube embed responsive so it doesn't run off the screen on mobile? [How-to link](https://www.h3xed.com/web-development/how-to-make-a-responsive-100-width-youtube-iframe-embed)